po/Makefile.am

```diff
@@ -66,7 +66,7 @@ po/POTFILES.html.in:
 	( cd $(srcdir) && find $(TRANSLATE) -name '*.html' ) > $@
 po/POTFILES.js.in:
 	$(MKDIR_P) $(notdir $@)
-	( cd $(srcdir) && find $(TRANSLATE) -name '*.js' -o -name '*.jsx' ) > $@
+	( cd $(srcdir) && find $(TRANSLATE) ! -name 'test-*' -name '*.js' -o -name '*.jsx' ) > $@
```

Comment:
Why do we mark strings for translation in tests in the first place?

Reply:
Because pkg/base1/test-locale.js is testing the very `_()` API :)

Reply:
Ah, of course :)